src/store/pages.js

```diff
@@ -93,23 +93,23 @@ export default {
 			return `/${pagePath}?fileId=${id}`
 		},
 
-		pagePathTitle: (_state, getters) => (page) => {
+		pagePathTitle: (_state) => (page) => {
```

Comment:
If you're neither using state nor getters you can leave out the parameters entirely.

```suggestion
		pagePathTitle: () => (page) => {
```

Reply:
Indeed, missed that one!